src/olympia/addons/models.py

```diff
@@ -352,6 +352,7 @@ class Addon(OnChangeMixin, ModelBase):
 
     is_experimental = models.BooleanField(default=False,
                                           db_column='experimental')
+    reputation = models.SmallIntegerField(default=0, null=True)
```

Comment:
We could, but then the database wouldn't match the django models anymore... (django's `default=` is different, it's handled entirely in python)